app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+
+  # def current_user?
```

Comment:
Will need something along the lines of this for nav bar to switch log in to log out and displau user name.